### DISCUSSION TITLE
Order of execution (first clangsa then clang-tidy)

### DISCUSSION THREAD

**Discussion by matthijs** [Ericsson/codechecker#4361] [General]
I am using codechecker in our project and now it seems to run clang-tidy / clangsa in random order. I want it to start with clangsa. But I could not find an option in the documentation to achieve that.

Some of our cpp files take about 90 minutes or more for clangsa to analyze. On a multicore machine it can start clangsa right from the beginning and the other cores can take care of the other files to analyze. In that way I can reduce the time it takes to analyze the project (ultimate goal).

Regards, Matthijs

**Comment by bruntib** [comment]
Hi @matthijs,

No, unfortunately the is no such feature in CodeChecker. The suggestion is reasonable, tough, so it would be worth creating a ticket on the issue.

**Reply by matthijs** [reply]
Ah too bad, thanks for the answer, I've created an issue: https://github.com/Ericsson/codechecker/issues/4362
